airbyte-webapp/src/views/Connection/ConnectionForm/components/SyncCatalogField.module.scss

```diff
@@ -1,33 +1,35 @@
-.headerCell {
-  font-size: 10px;
-  line-height: 13px;
-}
+@use "../../CatalogTree/CatalogTree.module.scss";
 
 .checkboxCell {
-  @extend .headerCell;
+  @extend %headerCell;
```

Comment:
headerCell was the same as in CatalogTree

Reply:
I think we might nevertheless not want to cross import from a completely different component (and not a variables file). I'd suggest we leave that duplicate for now, and clean it up as part of aligned text styling.